code/game/objects/structures/electricchair.dm

```diff
@@ -48,10 +48,10 @@
 	s.set_up(12, 1, src)
 	s.start()
 	if(buckled_mob)
-		buckled_mob.burn_skin(85)
+		buckled_mob.electrocute_act(85, src, 1)

```

Comment:
This doesn't get stopped by gloves, the insulated gloves stuff is handled in power code and not in `electrocute_act()`. You don't need to set any other arguments, this is fine. You just don't need the 1 at the end because that's the default value, but it's not really that big of a deal.